### PR TITLE
fix broken path for translation svg flags

### PR DESCRIPTION
**Pull request by diegowifi**
The flags icons were not showing because the assets path is wrong.

## Before

Screen Shot 2020-10-05 at 8 47 43 PM

## After

Screen Shot 2020-10-05 at 8 50 10 PM

### CODE REVIEW

**Comment by opdev1004** [comment]
I see that you updated file names. But is it necessary to add nan package?

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/react-fontawesome": "^0.1.9",
     "highlight.js": "^9.18.1",
+    "nan": "^2.14.1",
```
Comment (opdev1004) [inline package.json:17]: I see that you updated image file names. But is it necessary to add nan package?

**Review by opdev1004 — COMMENTED** [review]
What is nan package for?

File: package.json (hunk 1 above)
Comment (diegowifi) [reply package.json:17]: When I was running `npm install` it showed an error saying **_Cannot find module 'nan'_**, so I installed it to make it run, I'm not sure if it's missing in the package.json but I can remove it if necessary

Comment (diegowifi) [reply package.json:17]: This is the error message

Screen Shot 2020-10-05 at 7 34 08 PM

Comment (opdev1004) [reply package.json:17]: What OS are you using?

Comment (diegowifi) [reply package.json:17]: macOS Catalina 10.15.6

Comment (opdev1004) [reply package.json:17]: I found better solution! 🥇  You can install nan globally. And it won't affect this project and you won't have problem with other project.

`
npm i -g nan
`

Have a look at: [Error: Cannot find module 'nan'](https://stackoverflow.com/questions/34464673/error-cannot-find-module-nan)

Try it and have a look 😃

Comment (diegowifi) [reply package.json]: You are right, it worked 🙌🏻

I removed the dependency from _package.json_

Thanks 😃

**Pull request merged**